### PR TITLE
Update the nuttx-stm32f4 target

### PR DESCRIPTION
**Pull request by bzsolt**
JerryScript-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
Please add the related issue number to the commit message:

> Related issue: #1202

File: targets/nuttx-stm32f4/README.md [hunk 1]
```diff
@@ -1,143 +1,91 @@
...
+#### 3. Configure NuttX
 
-#### 3. Build JerryScript for NuttX
-
-```
-# assume you are in harmony folder
-cd jerryscript
-make -f ./targets/nuttx-stm32f4/Makefile.nuttx
 ```
+# assume you are in jerry-nuttx folder
```
Comment (zherczeg) [inline targets/nuttx-stm32f4/README.md]: I think these are assuming, not assume

File: targets/nuttx-stm32f4/README.md [hunk 2]
```diff
@@ -1,143 +1,91 @@
...
-
-#### 2. Prepare NuttX
-
-Follow [this](https://bitbucket.org/seanshpark/nuttx/wiki/Home) page to get
-NuttX source and do the first build. When it stops with and error,
-change default project from `IoT.js` to `JerryScript` as follows;
-
-2-1) run menuconfig
+  + apps
+  + stlink
```
Comment (zherczeg) [inline targets/nuttx-stm32f4/README.md]: Perhaps we should order them alphabetically.

**Comment by zherczeg** [comment]
LGTM after some minor changes

File: targets/nuttx-stm32f4/jerry_main.c [hunk 3]
```diff
@@ -272,10 +276,58 @@ jerry_port_log (jerry_log_level_t level, /**< log level */
...
+
+/**
+ * Dummy function to get the time zone.
+ *
+ * @return false
+ */
+bool
+jerry_port_get_time_zone (jerry_time_zone_t *tz_p)
+{
+  return false;
```
Comment (LaszloLango) [inline targets/nuttx-stm32f4/jerry_main.c]: I'd set tz_p->offset and tz_p->daylight_saving_time to 0 just to be sure.

**Comment by LaszloLango** [comment]
LGTM

**Pull request merged**